runtime/startup_byt.c

```diff
@@ -273,53 +273,92 @@ static char_os * read_section_to_os(int fd, struct exec_trailer *trail,
 
 */
 
+static void do_print_help(void)
+{
+  puts("Usage: ocamlrun [<options>] [--] <executable> [<command-line>]\n"
+       "Options are:\n"
+       "  -b  Set runtime parameter b (detailed exception backtraces)\n"
+       "  -I <dir>  Add <dir> to the list of DLL search directories\n"
+       "  -m  Print the magic number of <executable> and exit\n"
+       "  -M  Print the magic number expected by this runtime and exit\n"
+       "  -p  Print the names of the primitives known to this runtime\n"
+       "  -t  Increments the trace level in debug mode by 1\n"
```

Comment:
This is about tracing the bytecode interpreter.  Plus, "by 1" is slightly pedantic :-) 
```suggestion
       "  -t  Trace the execution of the bytecode interpreter. Multiple -t increase verbosity\n"
```

Reply:
It's ~probably~certainly just me, but the lack of the plural on "-t" bothers me... does "Trace the execution of the bytecode interpreter (specify multiple times to increase verbosity)" work?

Reply:
Yes, it works fine.